### ISSUE TITLE
add support PostCSS 5.x

### ISSUE THREAD

**Issue by DenisIzmaylov** [css-modules/css-modules-require-hook#25]
Hi! `npm` could not install this package to my project with [postcss 5.0.3](https://github.com/postcss/postcss/blob/master/package.json):
```sh
npm ERR! peerinvalid The package postcss@5.0.3 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer css-modules-require-hook@1.0.4 wants postcss@4.x
```

**Comment by sullenor** [comment]
Hi, I tried to update it to the 5 version yesterday, but looks like plugins should be updated first :)
So I downgraded it :)

Anyways, all tests pass with the version 5 and if you don't mind about warnings - you can use it.

**Comment by sullenor** [comment]
I'll link it to that Issue https://github.com/css-modules/css-modules-loader-core/issues/21

**Comment by sullenor** [comment]
Thank you, @joeybaker
I have published it in [1.0.5](https://github.com/css-modules/css-modules-require-hook/releases/tag/1.0.5)

**Comment by DenisIzmaylov** [comment]
Thank you
